CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contribute To Solr
+
+Thank you for your interest in contributing to the Apache Solr project. Solr continues to improve via the many contributions of developers around the world and new contributions are welcomed and encouraged.
+
+Your suggestions, comments and comments on Doris can be made directly through the [Apache Solr JIRA](https://issues.apache.org/jira/projects/SOLR/issues).
+
+There are many ways to participate in and contribute to Solr, such as: code implementation, test implementation, documentation improvements.
+
+If you have any questions along the way, you can reach out to the community using the resources provided [here](https://solr.apache.org/community.html#mailing-lists-chat).
+
+## How To Contribute
+
+Information on how to contribute can be found [here](https://github.com/apache/solr/blob/main/dev-docs/how-to-contribute.adoc).
```

Comment:
Instead of 'here' links, try to spell it out. Also, can the adoc link be relative and still work on github.com?
```suggestion
Information on how to contribute can be found in [dev-docs/how-to-contribute.adoc](https://github.com/apache/solr/blob/main/dev-docs/how-to-contribute.adoc).
```